MapleServer2/Packets/SkillDamagePacket.cs

```diff
@@ -56,8 +56,8 @@ public static PacketWriter Damage(SkillCast skillCast, int attackCount, CoordF p
         pWriter.Write(Mode.Damage);
         pWriter.WriteLong(skillCast.SkillSn);
         pWriter.WriteInt(attackCount);
-        pWriter.WriteInt(skillCast.Caster.ObjectId);
-        pWriter.WriteInt(skillCast.Caster.ObjectId);
+        pWriter.WriteInt(skillCast.Caster?.ObjectId ?? 0);
+        pWriter.WriteInt(skillCast.Caster?.ObjectId ?? 0);
```

Comment:
I wonder if it would be more beneficial and nicer to writer if all these bytewriters automatically write 0 if it's null.

Reply:
Agreed. I can go back and add an overload for things like WriteInt(int?)